sales/amazon_connector/setup.rst

```diff
@@ -2,16 +2,66 @@
 Configure Amazon Connector in Odoo
 ==================================
 
+The Amazon Connector uses an Authorization Token that allows the Odoo Amazon
+platform to fetch data from your Amazon Seller Central account. This token can
+be obtained directly through Seller Central in a few clicks and will then need
+to be set up in your Odoo database configuration.
+
+Generate an Authorization Token in Seller Central
+=================================================
+
+Connect to your Seller Central account for the marketplace you initially signed
+up (e.g. if you created your account on Amazon Germany, go to
+`Amazon Seller Central for Germany <https://sellercentral.amazon.de>`_) with an administrator account.
+
+.. note::
+  You might need to use the *main* (or first) administrator account and not one
+  added subsequentely.
+
+In the main menu, select :menuselection:`Apps & Services --> Manage Your Apps`;
+in the page that displays, click on the ``Authorize new developper`` button:
+
+.. image:: ./media/seller_central_apps.png
+  :align: center
+
+Fill in the form depending on your Marketplace:
+
+.. image:: ./media/seller_central_form.png
+  :align: center
+
+For Canada, Mexico or the United States, use these values:
```

Comment:
```suggestion
If your seller account is registered in the **North America** region, use these values:
```
This is more \*robust\* imho. Some sellers could have created their account solely for the Brazil or Netherlands marketplaces. Even if we don't officially support them (but still a bit), this should be less confusing.
BTW, I looked for an official mention of the account's region on Seller Central but couldn't find any. Still, [the doc calls them as such](http://docs.developer.amazonservices.com/en_US/dev_guide/DG_Endpoints.html).